boofuzz/sessions.py

```diff
@@ -1117,6 +1120,10 @@ def transmit_normal(self, sock, node, edge, callback_data):
                 self._fuzz_data_logger.log_fail(msg)
             else:
                 self._fuzz_data_logger.log_info(msg)
+        except exception.BoofuzzTargetConnectionFailedError as e:
+            self._fuzz_data_logger.log_fail(str(e))
+        except exception.BoofuzzSSLError as e:
+            self._fuzz_data_logger.log_fail(str(e))
```

Comment:
The same as [here](https://github.com/jtpereyda/boofuzz/pull/349/files/febb8dd2297e31d9c69e3be30e651b71e430b558#r348180471) 😉

Reply:
Fixed in Commit fc997eb

Reply:
Fixed in Commit fc997eb